packages/cipherstash-proxy/src/postgresql/data/to_sql.rs

```diff
@@ -47,8 +49,10 @@ fn binary_to_sql(plaintext: &Plaintext) -> Result<BytesMut, Error> {
         Plaintext::Utf8Str(x) => x.to_sql_checked(&Type::TEXT, &mut bytes),
         Plaintext::JsonB(x) => x.to_sql_checked(&Type::JSONB, &mut bytes),
         Plaintext::Decimal(x) => x.to_sql_checked(&Type::NUMERIC, &mut bytes),
-        // TODO: Implement these
-        Plaintext::BigUInt(_x) => unimplemented!(),
+        Plaintext::BigUInt(x) => {
```

Comment:
Was this needed here for timestamp, or is this a separate implementation? If so, is this covered with a test case?